content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -23,6 +23,16 @@ RUM Auto-Instrumentation works by injecting a RUM SDK JavaScript scriptlet into
 
 After your applications have been instrumented, you can configure your RUM application in Datadog.
 
+## Limitations
+
+The available functionality has the following important limitations:
+
+- If serving compressed traffic, the Auto-Instrumentation method is not able to inject the JS scriptlet into the HTML traffic.
```

Comment:
**💬 suggestion:** ‏this contradicts the advice in the "Troubleshooting" section, "and configure NGINX to compress the content.". It looks like Nginx is able to serve compressed traffic AND inject RUM.

Maybe something like:

```suggestion
- If proxying compressed traffic, the Auto-Instrumentation method is not able to inject the JS scriptlet into the HTML traffic.
```

Reply:
You'r right @BenoitZugmeyer. I would advise to move all the reasons why RUM is not injected here (because it's more precise and better formatted) and reference the limitations in the troubleshooting section.